### DISCUSSION TITLE
Discussion: add an en-US locale for "native plugins/theme's"?

### DISCUSSION THREAD

**Discussion by NekoJonez** [GlotPress/GlotPress-WP#1560] [General]
This is something I'd love to hear opinions about but there are plugins like: https://wordpress.org/plugins/huurkalender-wp/ 
which aren't in English. Of course, most of these plugins are more local plugins ment for the navite speakers.
But, what if people want to translate them in en-US, the default language of WP? 

I personally think that to achieve this, a plugin/theme should have some sort of tag that the language of the plugin isn't English but e.g. Dutch (in the example) and then it should appear underneath en-US as possible to translate. 

This is me just thinking out loud, and maybe I'm thinking too much about too niche plugins/themes... But, I just want to open the discussion about it.

**Comment by ocean90** [comment]
GlotPress already supports [en_US as a locale](https://github.com/GlotPress/GlotPress/blob/948010de22c673d388c6c6be08e33d8faf06c0db/locales/locales.php#L709-L717). I guess what you meant to request is whether translate.wordpress.org should get such a locale. For that I have converted this to a proper discussion even if this doesn't really belong here.

**Comment by NekoJonez** [comment]
Then I shall make a meta trac ticket.

**Reply by ocean90** [reply]
Or comment on existing ones like https://meta.trac.wordpress.org/ticket/3369.

**Reply by NekoJonez** [reply]
You saved me a search. Thanks. :)
